packages/editor/src/client/blocks/linkPageBreakBlock.tsx

```diff
@@ -11,44 +32,174 @@ export function LinkPageBreakBlock({
+          placeholder="Title"
+          label="Title"
+          style={{fontSize: '24px'}}
+          value={text}
+          disabled={disabled}
+          onChange={e => onChange({...value, text: e.target.value})}
+        />
+      </Box>
       <Box flexGrow={1}>
         <TypographicTextArea
           ref={focusRef}
-          variant="h2"
-          placeholder="Text"
-          align="center"
-          value={text}
+          variant="body1"
+          placeholder="Optional Text"
+          align="left"
+          value={htmlText}
+          onChange={e => onChange({...value, htmlText: e.target.value})}
+        />
+      </Box>
+      <Box style={{width: '50%', display: 'inline-block'}}>
+        <TextInput
+          ref={focusInputRef}
+          placeholder="Link Text"
+          label="Button label"
+          value={linkText}
           disabled={disabled}
-          onChange={e => onChange({...value, text: e.target.value})}
+          onChange={e => onChange({...value, linkText: e.target.value})}
+        />
+      </Box>
+      <Box style={{width: '50%', display: 'inline-block', padding: '10px'}}>
+        <TextInput
+          ref={focusInputRef}
+          placeholder="Link URL"
+          label="Button link"
+          value={linkURL}
+          disabled={disabled}
+          onChange={e => onChange({...value, linkURL: e.target.value})}
         />
       </Box>
-      <TypographicTextArea
-        ref={focusRef}
-        variant="body1"
-        placeholder="Link Text"
-        align="center"
-        value={linkText}
-        disabled={disabled}
-        onChange={e => onChange({...value, linkText: e.target.value})}
-      />
-      <TypographicTextArea
-        ref={focusRef}
-        variant="subtitle2"
-        placeholder="Link URL"
-        align="center"
-        value={linkURL}
-        disabled={disabled}
-        onChange={e => onChange({...value, linkURL: e.target.value})}
-      />
+      <br />
+      <br />
+      <div className={'option-wrapper'} style={{display: 'flex'}}>
+        <Card
+          overflow="hidden"
+          width={200}
+          height={150}
+          flexGrow={1}
+          marginRight={Spacing.ExtraSmall}>
+          <Box padding={'10'}>
+            <p>Colors and Styles</p>
+            <small>Styles: </small>
+            <select
+              defaultValue={styleOption}
+              onChange={e => onChange({...value, styleOption: e.target.value || ''})}>
+              <option value="default">Default Color</option>
+              <option value="light">Light Color Style</option>
+              <option value="dark">Dark Color Style</option>
+            </select>
+          </Box>
+          <Box padding={'10'}>
+            <small>Layouts: </small>
+            <select
+              defaultValue={layoutOption}
+              onChange={e => onChange({...value, layoutOption: e.target.value || ''})}>
+              <option value="default">Default Layout</option>
```

Comment:
A little bit the same issue as with the field above. Also i think you copy/pasted a bit too much.

If you want to have that field as a left, center or right aligned field then you should change the type in the API to only allow those values and then present here a Radio Group.

Reply:
MVP,  out of scope.

This good for now, there will be a final cleanup after merge of this PR => as said, this needs to be merge, that we can start 
to develop.